brainstormforce/ultimate-addons-for-gutenberg
# Fix: Social Share - Fixed jQuery undefined issue.

### Description

Trello - https://trello.com/c/VKry98uR/281-fix-social-share-fixed-jquery-undefind-issue

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Activate social share block and drag and drop on pages.
- Check out the console log for error
- Drag and drop TOC, Inline notice, Forms, Blockquote, Lottie and check everything is working fine on the frontend. No issue in the console.

### Checklist:
